offchain-modules/packages/x/src/ckb/tx-helper/generator.ts

```diff
@@ -613,13 +613,145 @@ export class CkbTxGenerator extends CkbTxHelper {
   }
 
   async lockSUDT(
```

Comment:
1. Forgot to handle user sudt change cell? If user want to lock 100 sudt, and inputs collect 120 sudt of user, then outputs should refund 20 sudt to user sudt change cell
2. Should calculate and handle tx fee

Reply:
1. sudt change with minimal cell capacity is written in the second output (if needed), ckb change with removal fee and sudt change cell capacity are written in the last output
2. add the calculation of the tx fee at the end